### PR TITLE
MWI restructure

### PR DESCRIPTION
**Pull request by Tinkaa**
Some restructuring of files

- Proposing to remove gee_example.ipynb since this has been implemented in src/utils_general/gee_zonal_stats_ts.py
- Proposing to remove globalipc_erroradm0aggr.ipynb as this was more relevant in Somalia, and no global IPC data was used for MWI 
- Proposing to remove mwi_seasonal_indicators, since the same parameters have been analysed more comprehensively in mwi_impact.R
- Moving the documentation to a separate folder since the notebook directory got very cluttered

More restructuring of removing all the `.ipynb's`, numbering, and cleaning up notebooks will follow but want to keep the PRs doable..

### CODE REVIEW

File: analyses/malawi/notebooks/mwi_arc2_identify_dry_spells.md [hunk 1]
```diff
@@ -1,25 +1,11 @@
...
----
-
 ## Compute dry spells detected by ARC2
 [ARC2](https://iridl.ldeo.columbia.edu/SOURCES/.NOAA/.NCEP/.CPC/.FEWS/.Africa/.DAILY/.ARC2/.daily/.est_prcp/) is a data set of daily observed precipitation data that is updated relatively quickly compared to other data sets. 
 
 This notebook retrieves the data, computes dry spells, and merges it with observed dry spells in the CHIRPS dataset in order to compare the two. 
 
-TODO:
-- Find a way to quickly incorporate new values, since computing on the whole dataset takes several hours
+Note that this notebook is in a very experimental state. However, since after exploration it was decided to not use ARC2 as a data source as of now, we don't invest time in improving this. The reason of not using ARC2 was because it is observational data and a preference was given for forecast data.
```
Comment (hannahker) [inline analyses/malawi/notebooks/mwi_arc2_identify_dry_spells.md:6]: Really useful to keep track of notes like these!

**Review by hannahker — APPROVED** [review]
Looks good to me, and good to do this kind of clean up! Happy to have this merged. I really like having a ```docs``` folder - we should make this a standard for each country. 

Some general structuring/organization ideas for future work: 
- We should work on drafting a mwi readme when both the dry spells and flooding work is relatively wrapped up. 
- For organizing ```.md``` files, I like what Monica's done in numbering files to keep things generally in a logical order for an analysis. Maybe worth implementing here?
- I've been thinking that it's somewhat redundant to add the iso3 prefix when naming scripts and notebooks, since everything is already organized in a single country file. What do you think? 
- After the flooding trigger work is merged we should also probably create a directory to keep all of this dry spells work separate.

**Comment by hannahker** [comment]
Ah I see you already did most of these in #118 😄

**Comment by Tinkaa** [comment]
Oops sorry that was maybe not very clear, but thanks a lot for these quick reviews!

**Pull request merged**